services/QuillLMS/engines/evidence/app/views/evidence/research/gen_ai/llm_trial_suites/new.html.erb

```diff
@@ -52,6 +67,18 @@
   </div>
 
   <script>
+    function filterByAiType(filter) {
+      const sections = document.querySelectorAll('.ai-type-section');
+
+      sections.forEach(section => {
+        if (filter === 'all' || section.dataset.aiTypeSection === filter) {
+          section.style.display = 'block';
+        } else {
+          section.style.display = 'none';
+        }
+      });
+    }
```

Comment:
This feels weird.  I'd take Emilia's opinion on this more than mine, but it feels like this should be some sort of CSS class that gets conditionally assigned to the `<section>` elements (whatever they actually are) based on this condition.